mkdocs.yml

```diff
@@ -656,11 +657,11 @@ nav:
           - Controllers: "development/core/controllers.md"
           - Generic Views: "development/core/generic-views.md"
           - Home Page Panels: "development/core/homepage.md"
-          - Local Kubernetes Cluster: "development/core/local-k8s.md"
           - Model Development Checklist: "development/core/model-checklist.md"
           - Model Features: "development/core/model-features.md"
           - Natural Keys: "development/core/natural-keys.md"
           - Navigation Menu: "development/core/navigation-menu.md"
+          - Kubernetes Job Support: "development/core/kubernetes-job-support.md"
```

Comment:
Looks like this section was loosely in alphabetical order, so maybe move this up a few lines?